.github/workflows/ci_workflow.yml

```diff
@@ -69,8 +69,10 @@ jobs:
     - name: Run PyLint
       run: pylint --rcfile=pyproject.toml *.py tools
 
+    # Ignore ID 44715 for now.
+    # See this NumPy issue for more information: https://github.com/numpy/numpy/issues/19038
     - name: Run safety
-      run: pip freeze | safety check --stdin
+      run: pip freeze | safety check --stdin --ignore 44715
```

Comment:
From the last test it seems that safetly fails on both 44716 and 44717.

Reply:
Yes, and also 44715. See my [comment above](https://github.com/emmo-repo/EMMO-python/pull/361#issuecomment-1027786041) for what I suggest for 44716 and 44717.

Reply:
Ok, I think I misunderstood you then.  I thought you had opened a finshed PR. I agree with ignoring them, if what they say about this not being a real issue is correct.

Reply:
Yeah - it was more of an open question - because we could also go the route of dropping Python 3.6 and 3.7 support - but that seems a bit drastic.

Reply:
I agree, that is abit drastic in my eyes too.